test/integration/01-tx-send.js

```diff
@@ -139,7 +138,7 @@ describe('Sending transactions', () => {
 
       it('should fail when signing request times out', async () => {
         const amount = 100;
-        const txSigningDelay = config.txSigningTimeout + 10; // set delay above the allowed timeout
+        const txSigningDelay = process.env.TX_SIGNING_TIMEOUT + 10; // set delay above the allowed timeout
```

Comment:
Can you set a default for this in the test file in case the test is called without setting this environment variable?

Reply:
Unfortunately no, calling test without this env var will break sign timeout checks.

Reply:
I mean can you not put at the top of the file:

```suggestion
        const txSigningTimeout = process.env.TX_SIGNING_TIMEOUT || 100
```

So that it is configurable but not vital

Reply:
I feel like we are duplicating too many configuration values as environment variables. Could you not pass a value for txSigningTimeout into marketplaceTx at line 23 and just use that value here?